# Reset the encoder for the given node and motor

uint8 node 
uint8 channel
uint32 value 0
---
bool success
string error "success"